docs/contributing/index.md

```diff
@@ -196,8 +196,9 @@ These branches may change as we get closer to a production release.
 
     ```shell
     # choose one
-    cd submodules/plone.restapi
     cd submodules/volto
+    cd submodules/plone.restapi
+    cd submodules/plone.api
```

Comment:
Let's sort alphabetically. I imagine we might import more projects.

Reply:
My thoughts why I made a resort: People that are not familiar with plone.restapi and plone.api do mix them up from time to time. If they are mentioned side by side, it's obvious that there exist two different packages.

Reply:
The alpha sort would be:

```
cd submodules/plone.api
cd submodules/plone.restapi
cd submodules/volto
```

...which coincidentally accomplishes the same intent.